crates/forge/src/cmd/compiler.rs

```diff
@@ -93,11 +93,18 @@ impl ResolveArgs {
                         .collect();
 
                     let evm_version = if shell::verbosity() > 1 {
-                        Some(
-                            EvmVersion::default()
-                                .normalize_version_solc(version)
-                                .unwrap_or_default(),
-                        )
+                        let evm = EvmVersion::default()
+                            .normalize_version_solc(version)
+                            .unwrap_or_default();
+
+                        // Vyper does not yet support Prague, so we normalize it to Cancun.
+                        if matches!(language, MultiCompilerLanguage::Vyper(_)) &&
```

Comment:
not a blocker, but can we add `fn is_vyper` functions to this type?

Reply:
Adding in https://github.com/foundry-rs/compilers/pull/276